### PR TITLE
Mark an `additionalProperty` as evaluated.

### PR DESCRIPTION
**Pull request by davidmorgan**
## Ultimate problem:

Reported in #190, a property should count as evaluated if it's an `additionalProperty`.

## How it was fixed:

Call `_addEvaluatedProp` when doing validation of an `additionalProperty`.

## Testing suggestions:

Added a test.

## Potential areas of regression:

Are there cases where an additional property is validated where it _shouldn't_ count as evaluated?

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Comment by davidmorgan** [comment]
@joecotton-wk hi Joe, it looks like you're the last person committing/maintaining `json_schema`, could I ask if someone is able to accept / review PRs, please? :) just to get an idea if this is likely to be accepted. Thanks!

This package looks really useful and well tested, seems to be just a one line fix to get my use case working. I hope :)

**Comment by joecotton-wk** [comment]
@davidmorgan Hello 👋 I will get someone to review and take care of this in the next business day or so 👍

**Review by michaelcarter-wf — DISMISSED** [review]
+1

**Review by michaelcarter-wf — APPROVED** [review]
+1

**Comment by michaelcarter-wf** [comment]
@davidmorgan it looks like I also needed to approve your CI run as an external contributor. I've done that and it is now running. We'll try to get this merged today. Thanks for your contribution!

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Comment by davidmorgan** [comment]
Thanks! I also sent #189 but that's just an incidental fix, not something I'm blocked on :)

**Pull request merged**
